.github/dependabot.yml

```diff
@@ -1,24 +1,132 @@
 version: 2
 updates:
-
+  # =============================================================================
+  # GITHUB ACTIONS DEPENDENCIES
+  # =============================================================================
+  # All GitHub Actions grouped together - these are generally safe to auto-merge
+  # since they don't affect runtime behavior and are validated by CI
   - package-ecosystem: "github-actions"
     directory: "/"
-    open-pull-requests-limit: 50
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "monday"
+      time: "06:00"
+      timezone: "America/Chicago"
+    open-pull-requests-limit: 5
     labels:
       - "sdou"
       - "dependencies"
+      - "github-actions"
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+    ignore:
+      - dependency-name: "actions/delete-package-versions"
+        update-types: ["version-update:semver-major"]
 
+  # =============================================================================
+  # MAVEN PRODUCTION DEPENDENCIES
+  # =============================================================================
+  # Core runtime dependencies that affect production behavior
+  # These require manual review due to potential impact on end users
   - package-ecosystem: "maven"
     directory: "/"
-    open-pull-requests-limit: 50
     schedule:
-      interval: "daily"
+      interval: "weekly"
```

Comment:
I think this one could remain daily, as it may bring security dependabots for production dependencies.  What do you think?

Reply:
@filipelautert fixed in https://github.com/liquibase/liquibase/pull/7106/commits/547e7b6d5af9ff78dd9c751979734d28599dedba